CHANGELOG.md

```diff
@@ -4,15 +4,20 @@ All notable changes to this project will be documented in this file.
 
 ## [4.5.0] - TBD
 
-Wazuh commit: TBD
+Wazuh commit: TBD \
```

Comment:
Typo: backslash at the end should not exist

Reply:
Fixed https://github.com/wazuh/wazuh-qa/pull/4307/commits/4e75c6ef578b50d2f296da490e3b98b0c93ff1c2

<details><summary>Image</summary>

![image](https://github.com/wazuh/wazuh-qa/assets/14913942/3bfd586b-2231-4d89-8c42-60c4e3c9818e)

</details>